includes/admin/class-wcmp-export.php

```diff
@@ -648,13 +702,12 @@ public function prepareReturnShipmentData($order_id, $options = [])
     public static function getRecipientFromOrder(WC_Order $order)
     {
         $isUsingMyParcelFields = WCX_Order::has_meta($order, "_billing_street_name")
-                                 || WCX_Order::has_meta($order, "_billing_house_number");
+            || WCX_Order::has_meta($order, "_billing_house_number");
```

Comment:
Kijk of je deze check, gewoon op 1 regel kan zetten

Reply:
Fixed